### PR TITLE
More Agoric tweaks

### PR DESCRIPTION
**Pull request by michaelfig**
I found these changes were useful to help make the Agoric dynamic IBC layer work correctly:

- allow `rly config show --json` to dump the YAML config as JSON
- `rly tx conn` should create the client if it isn't already
- minor fix to add a missing help `%s` format argument to `rly tx raw send`
- have `nchainz` default to set up the connections but not relay any paths
- turn down the Agoric chain `--log_level` to `warn` to help prevent noise

### CODE REVIEW

**Review by colin-axner — APPROVED** [review]
utACK, thanks @michaelfig

File: cmd/tx.go [hunk 1]
```diff
@@ -206,7 +206,15 @@ $ %s tx con demo-path -o 3s`, appName, appName, appName)),
 				return err
 			}
 
-			modified, err := c[src].CreateOpenConnections(c[dst], retries, to)
+			// ensure that the clients exist
+			modified, err := c[src].CreateClients(c[dst])
```
Comment (AdityaSripal) [inline cmd/tx.go:210]: Should similar logic exist for `CreateChannel`? Where we create client and connection if they don't exist?

cc: @colin-axner

**Review by AdityaSripal — APPROVED** [review]
utACK, thanks @michaelfig!

File: cmd/tx.go (hunk 1 above)
Comment (michaelfig) [reply cmd/tx.go:210]: > Where we create client and connection if they don't exist?

Possibly.  The current `tx link` subcommand does that already, FWIW.

Comment (colin-axner) [reply cmd/tx.go:210]: no preference here, maybe it makes sense to deprecate `create channels`? or rename `tx link`? Doesn't make too much sense to maintain 2 commands that do the same thing

Comment (colin-axner) [reply cmd/tx.go:210]: opened #400, outside the scope of this pr

**Pull request merged**